cds-snc/cppd-medical-report
# Memory session

# Description

Replace the cookie session store with an in memory session store

# Test Instructions

1. Enter in some data on personal
1. navigate to confirmation 
1. profit and by profit I mean ensure that the data shows up.
